by-language/scala-spark-http/README.md

```diff
@@ -0,0 +1,24 @@
+# Scala example loading data from Spark to CrateDB via http
+
+This example aims to demonstrate how a Spark data frame can be loaded into CrateDB using the CrateDB http endpoint.
+
+It assumes there is a CrateDB instance running on localhost accepting connections with the default `crate` superuser, and it relies on the following table being created:
+
+.. code-block:: sql
+
+    CREATE TABLE myschema.mytable (
+        examplefield1   TIMESTAMP,
+        anotherfield    geo_point,
+        jsonpayload     OBJECT
+    );
+
+In environments with very high volumes of data you may want to consider replacing scalaj.http with async calls with `akka.http` or `AsyncHttpClient`.
```

Comment:
I would not necessarily connect "high volumes of data" with async/non-blocking i/o matters. The right choice depends much more on the types of workloads at hand. [TLDR; Making the Right Choice about multiprocessing vs multithreading vs asyncio](https://stackoverflow.com/a/52498068) shares a concise summary -- while it's Python focused, general principles apply to Scala's and other runtimes as well.

```suggestion
When applicable for your environment, you may want to consider to replace `scalaj.http` with async calls like `akka.http` or `AsyncHttpClient`.
```